guest/enqueue/clusterevent.go

```diff
@@ -0,0 +1,42 @@
+package enqueue
+
+import "sigs.k8s.io/kube-scheduler-wasm-extension/guest/api"
+
+// minEncodedClusterEvent is the size in bytes to encode api.ClusterEvent with
+// 32-bit little endian gvk, ActionType, NUL-terminated Label,
```

Comment:
note: later we need to make an ABI doc with details like this consolidated, but as everything is in-flux not doing this right now and instead relying on tests.